### DISCUSSION TITLE
TC1797 support

### DISCUSSION THREAD

**Discussion by Carl-NRCan** [NationalSecurityAgency/ghidra#6332] [Ideas]
Hi !
Would it be possible to add the [TC1797.pspec](https://github.com/fanyi3315/ghidra_tc1797/blob/tc1797/Ghidra/Processors/tricore/data/languages/tc1797.pspec) file in a future release ? Thanks :)

**Comment by ecuprimetw** [comment]
is it for car ecu?

**Reply by ctrlCit** [reply]
yes
